apps/src/templates/progress/ProgressTable.jsx

```diff
@@ -32,26 +36,53 @@ const ProgressTable = React.createClass({
   },
 
   render() {
-    if (this.props.isSummaryView) {
+    const { isSummaryView, categorizedLessons, hasPeerReviews } = this.props;
+
+    const TableType = isSummaryView ? SummaryProgressTable : DetailProgressTable;
```

Comment:
I cant call this ProgressTable, as that would clash with the component name. Instead, I'll have the other usage also use TableType.